### ISSUE TITLE
[Bug]: Textbox doesn't accept input

### ISSUE THREAD

**Issue by yM4rvinz** [matplotlib/matplotlib#22874]
### Bug summary

Typing anything inside the textbox is not working for me. The textbox doesn't update graphically, on the plot figure, neither on its value, demonstred with "texto.on_text_change(lambda v: print(v))" on the code. However, if i press [tab] key once, the textbox works just fine and if i press it twice, it stops working again.
This issue isn't occuring only to me, because i've found it on this post: "https://serveanswer.com/questions/alternatives-to-matplotlib-textbox-widget", but with no solution or answers. In addition, i have also shared the code with some friends and it worked just fine for them.

### Code for reproduction

```python
import matplotlib.pyplot as plt
from matplotlib.widgets import TextBox

textoax = plt.axes([0.1, 0.5, 0.5, 0.1])
texto = TextBox(ax=textoax, initial="test", label="testing")
texto.on_text_change(lambda v: print(v))

plt.show()
```

### Actual outcome

No typing is occurring on the textbox.

https://user-images.githubusercontent.com/104156100/164502724-736bf5d9-78d9-4d4f-a9ae-12f3eef13da8.mp4

### Expected outcome

The textbox should update its value when typing something.

### Additional information

I've tried reinstalling matplotlib, no sucess.
Using matplotlib 3.5.0 version, no sucess.
Changing textbox properties as "initial" and "label" in multiple ways, no sucess.
Copying the entire code used in a textbox tutorial, and it also doesn't work.
Changing my keyboard configuration to English, rather than ABNT2 (Portuguse), no sucess.

### Operating system

Windows 10

### Matplotlib Version

3.5.1

### Matplotlib Backend

TkAgg

### Python version

3.9.9

### Jupyter version

None

### Installation

pip

**Comment by daniilS** [comment]
This is fixed by #22077. Until then, the simplest workaround is to press Tab to set the focus to the canvas first.

**Comment by timhoffm** [comment]
I think this is a duplicate to # 13484.
